### PR TITLE
copy to returning a stream

### PR DESCRIPTION
**Pull request by oleurud**
In order to use COPY TO with pandas it in CARTOframes (https://github.com/CartoDB/cartoframes/issues/563), we will need to add this feature. After some doubts (also other approaches), I think this is the best option, but let me know if you think the same.

After that, we will be able to do:
```python
response = copyClient.copyto_stream(query)
df = pandas.read_csv(response)
```

Please, I am beginner here, so any comment will be appreciated

### CODE REVIEW

File: carto/utils.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, response):
+        self.it = response.iter_content(DEFAULT_CHUNK_SIZE)
+        self.leftover = None
+
+    def readable(self):
+        return True
+
+    def readinto(self, b):
+        try:
+            lenght = len(b)
```
[the remark this answers is not in the document]
Comment (oleurud) [reply carto/utils.py]: how many times I have written it wrong

File: examples/copy_and_pandas_example.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import argparse
+import os
+import sys
+import logging
+import pandas as pd  # you will need to install it
```
[the remark this answers is not in the document]
Comment (oleurud) [reply examples/copy_and_pandas_example.py]: Ahhh oki :+1:

File: carto/sql.py [hunk 3]
```diff
@@ -454,3 +455,14 @@ def copyto_file_path(self, query, path, append=False):
         file_mode = 'wb' if not append else 'ab'
         with open(path, file_mode) as f:
             self.copyto_file_object(query, f)
+
+    def copyto_stream(self, query):
+        """
+        Gets data from a table into a stream
+        :param query: The "COPY { table_name [(column_name[, ...])] | (query) }
+                           TO STDOUT [WITH(option[,...])]" query to execute
+        :type query: str
```
Comment (javitonino) [inline carto/sql.py]: We need to document the return type. Something like:

```
:return: the data output from COPY
:rtype: file-like object
```

File: carto/utils.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+from io import RawIOBase
+
+DEFAULT_CHUNK_SIZE = 8 * 1024  # 8 KB
```
Comment (javitonino) [inline carto/utils.py]: This constant could be inside the class, since it only really applies to that class

File: carto/sql.py (hunk 3 above)
Comment (oleurud) [reply carto/sql.py]: :+1:

File: carto/utils.py (hunk 4 above)
Comment (oleurud) [reply carto/utils.py]: True!

**Pull request merged**